web/modules/custom/dpl_example_content/content/node/02858224-6bd4-41df-95e3-c2a9a6ad29b7.yml

```diff
@@ -0,0 +1,889 @@
+          behavior_settings:
+            -
+              value: {  }
+          revision_translation_affected:
+            -
+              value: true
+          field_filter_branches:
+            -
+              entity: dac275e4-9b8c-4959-a13a-6b9fdbc1f6b0
+          field_filter_cond_type:
+            -
+              value: or
+          field_max_item_amount:
+            -
+              value: '12'
+          field_title:
+            -
+              value: 'Event list - automatic'
+    -
+      entity:
+        _meta:
+          version: '1.0'
+          entity_type: paragraph
+          uuid: b26d1ea6-f15a-41e8-9746-af3818dd0637
+          bundle: card_grid_automatic
+          default_langcode: da
+        default:
+          status:
+            -
+              value: true
+          created:
+            -
+              value: 1719475285
+          behavior_settings:
+            -
+              value: {  }
+          revision_translation_affected:
+            -
+              value: true
+          field_filter_cond_type:
+            -
+              value: or
+          field_filter_content_types:
+            -
+              value: article
+          field_title:
+            -
+              value: 'Card grid automatic - artikler'
+    -
+      entity:
+        _meta:
+          version: '1.0'
+          entity_type: paragraph
+          uuid: c02a1e27-450f-4b26-bb95-d0cda8f4c15b
+          bundle: card_grid_manual
+          default_langcode: da
+        default:
+          status:
+            -
+              value: true
+          created:
+            -
+              value: 1719475399
+          behavior_settings:
+            -
+              value: {  }
+          revision_translation_affected:
+            -
+              value: true
+          field_grid_content:
+            -
+              entity: 2cd0fe5e-4159-4452-86aa-e1a1ac8db4a1
+            -
+              entity: 982e0d87-f6b8-4b84-8de8-c8c8bcfef557
+            -
+              entity: fd774222-4b22-4803-ae45-2d4e80d84fac
+            -
+              entity: 479ad4da-b402-4eab-801f-0d3db375df1c
+            -
+              entity: 9cfd15df-32b4-4af9-b2da-56b039b5bf6b
+            -
+              entity: 33a56f4a-fa42-42e1-9097-84f759bbbe4c
+          field_title:
+            -
+              value: 'Card grid manual'
+    -
+      entity:
+        _meta:
+          version: '1.0'
+          entity_type: paragraph
+          uuid: ef0d89bf-7ea2-4881-bfc2-82feb3ae2e96
+          bundle: breadcrumb_children
+          default_langcode: da
+        default:
+          status:
+            -
+              value: true
+          created:
+            -
+              value: 1719478236
+          behavior_settings:
+            -
+              value: {  }
+          revision_translation_affected:
+            -
+              value: true
+          field_title:
+            -
+              value: 'Breadcrumb paragraph - Bestsellers'
```

Comment:
It seems like this is missing a value for the referenced breadcrumb item.

Reply:
Yep that's true. I forgot to readd in the process of removing the use of breadcrumb example content